tests/integration.rs

```diff
@@ -128,14 +127,12 @@ fn assert_advisory_output(output: &std::process::Output, expected_advisories: Ve
     assert_eq!(expected, actual_advisories);
 }
 
-#[test]
-#[serial]
+#[allow(dead_code)]
```

Comment:
TODO: translate these tests to use the Abscissa acceptance testing functionality